tsconfig.json

```diff
@@ -6,7 +6,7 @@
     "esModuleInterop": true,
     "forceConsistentCasingInFileNames": true,
     "isolatedModules": true,
-    "jsx": "preserve",
+    "jsx": "react",
```

Comment:
Why did you change it?

Reply:
that was automatically changed after running updated react-scripts